### PR TITLE
chore: fb_custom_audience v1 tests

### PR DESCRIPTION
**Pull request by sandeepdsvs**
## What are the changes introduced in this PR?

fb_custom_audience v1 tests

## What is the related Linear task?

Resolves INT-1482

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

File: test/integrations/destinations/fb_custom_audience/dataDelivery/business.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 200,
+            message: 'Request Processed Successfully',
+            response: [
```
Comment (koladilip) [inline test/integrations/destinations/fb_custom_audience/dataDelivery/business.ts:71]: what happens to stats for such errors?

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3095/test-report.html

**Review by ItsSudip — DISMISSED** [review]

File: test/integrations/destinations/fb_custom_audience/dataDelivery/other.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message: 'There have been too many calls to this ad-account.',
+            statTags: {
```
Comment (koladilip) [inline test/integrations/destinations/fb_custom_audience/dataDelivery/other.ts:45]: can we use common statTags and override only fields that are changed for this test case using { ... }

**Review by koladilip — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**